purchase_order_line_description/models/res_config_settings.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Alex Comba - Agile Business Group
+# Copyright 2017 Tecnativa - vicent.cubells@tecnativa.com
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _name = "res.config.settings"
+    _inherit = "res.config.settings"
+
+    group_use_product_description_per_po_line = fields.Selection(
+        selection=[
+            ("0", "Use the product name on the purchase order lines"),
+            ("1", "Use the product purchase description on the purchase" "order lines"),
```

Comment:
Just the comment to "Black". I am not sure why it create space between purchase and order. :p